package.json

```diff
@@ -10,7 +10,7 @@
     "doctoc": "doctoc --maxlevel 3 README.md",
     "jsdoc:widget": "node ./scripts/doc/gen-widget-doc.js",
     "gh-pages": "./scripts/gh-pages.sh",
-    "lint": "eslint . --quiet --no-color && sass-lint --verbose",
+    "lint": "eslint ./components ./decorators ./lib ./widgets --quiet --no-color && sass-lint --verbose",
```

Comment:
If some file is failing and we want it to use var then we can just ignore it but our tests should be linted